### ISSUE TITLE
[Bug Report]: 

### ISSUE THREAD

**Issue by isfang** [Ideolys/carbone#110]
**Environment**
Carbone Version: carbone@2.1.1
Node Version: v14.12.0
Browsers: --
Desktop OS: Mac Os 11.0.1 (20B29)
LibreOffice: tried 7.0.4 & 6.4.7

**Expected Behaviour**

1. Template: use a 'docx' file with a pie chart 
2.Just inject a simple json data into the template.The 'result.docx' hopes to be displayed normally.

**Actual Behaviour**
After the data is replaced, the pie chart cannot be displayed normally in 'result.docx'.

**Steps to reproduce**
Steps to reproduce the bug:
1. git clone https://github.com/isfang/carbone-research.git
2. cd carbone-research
3. node test.js
4. open 'result.docx', the pie chart can not be displayed normally

**Screenshots**
template file:
![image](https://user-images.githubusercontent.com/12061812/106101727-4581be80-6179-11eb-9025-58ebeb9a39b3.png)
after data replace:
![image](https://user-images.githubusercontent.com/12061812/106101804-62b68d00-6179-11eb-822a-3550170160f0.png)

but 'result.docx' can not be displayed normally.

**Reproduction Link / Code example**
You can share your Carbone code through https://codesandbox.io/ or directly on this file.

**Other comments**

**Comment by isfang** [comment]
If you do not add 'hardRefresh: true', edit and close the result file it will display normally. If you add it, it will also be invalid.

**Comment by steevepay** [comment]
Hello @isfang, thank you for reaching us!
Carbone is not able yet to generate DOCX reports with charts and this is a project coming for 2021. I keep your use case for future testing ;)

**Comment by isfang** [comment]
> Hello @isfang, thank you for reaching us!
> Carbone is not able yet to generate DOCX reports with charts and this is a project coming for 2021. I keep your use case for future testing ;)

Thank you for your reply, maybe 'embeddings/Microsoft_Excel____\d' has been processed,  '\/word/charts\/chart[0-9]*.xml also ' needs to be processed.

**Comment by steevepay** [comment]
Yes, I already analyzed and noted that part for future development ;)

**Comment by BeaG130** [comment]
Charts have been supported since v4.
